### DISCUSSION TITLE
Feature request: inverted pen color for export on documents with black background

### DISCUSSION THREAD

**Discussion by liakhovitch** [flxzt/rnote#498] [Ideas]
When exporting a file that has a dark background, it would be nice to have an "invert pen color" option along with the "remove background" option. This helps avoid the following scenario:
- I work on a homework assignment with a black background and white pen in order to save my eyes.
- When I export as a PDF so I can submit, I check "remove background".
- The exported file is now an all-white page

I don't know how OneNote does this, but there are a few variations on how this could work:
- Just swap any color exactly matching "black" and "white" on the default color palette (probably good enough for most cases)
- Invert the value of any color below a certain saturation

This would cover pretty much all of _my_ use cases, but it would also be nice to have the option to do this when switching backgrounds. Having an option to automatically invert both the background color and pen colors when switching between light/dark mode would also be helpful.

**Comment by flxzt** [comment]
continuing in #635
